docs/SelectionToolbar.md

```diff
@@ -0,0 +1,39 @@
+# Selection Toolbar
+
+The `<pxb-selection-toolbar>` component is used to display a a toolbar with a selection list. 
+
+## API
+
+Parent element (`<pxb-selection-toolbar>`) attributes:
+
+<div style="overflow: auto;">
+
+| @Input   | Description                                    | Type                 | Required | Default |
+| -------- | ---------------------------------------------- | -------------------- | -------- | ------- |
+| title    | The text to display for title                  | `string`             | no       |         |
+| subtitle | The text to display selection                  | `string`             | no       |         |
+
+</div>
+
+The following child elements are projected into `<pxb-selection-toolbar>`:
+
+| Selector          | Description                                  | Required | Default |
+| ----------------- | -------------------------------------------- | -------- | ------- |
+| pxb-icon          | Icon shown on the left                       | no       |         |
+| pxb-menu          | Content to be shown within dropdown menu     | no       |         |
+| pxb-right-content | Content to be shown on right side of toolbar | no       |         |
+
+### Classes
+
+Each PX Blue component has classes which can be used to override component styles:
+
+| Name                                        | Description                            |
+| ------------------------------------------- | ---------------------------------------|
+| pxb-selection-toolbar                       | Styles applied to the root element     |
+| pxb-selection-toolbar-icon-wrapper          | Styles applied to the left icon        |
+| pxb-selection-toolbar-title-wrapper         | Styles applied to the title wrapper    |
```

Comment:
Can you update these to be different names.

This is the same naming convention we use when we project content.   When I read this, I expected title and subtitle to be supplied via [ng-content].

Reply:
We're currently using this class naming convention in all of our components. Did you have another naming convention in mind, and if so do we want to update all of our other components?

Reply:
If the content is being projected in and we have a wrapper class for it, it should be using this convention: 
`pxb-[component-name]-[content-name]-wrapper`.

If it's wrapping an element within the component, without any content project maybe we should stick to using this as a standard:
`pxb-[component-name]-[element-class]-container`

I would not go back and update any other component as part of this PR. 

Any class names we make will be considered breaking changes; I would like to add an item to revist our existing components and update our class names if this proposal makes sense.